### ISSUE TITLE
Local LLM

### ISSUE THREAD

**Issue by ruheil** [rihal-om/rihal-codestacker#3]
Is the use of a local LLM (on-prem) allowed?

**Comment by Shidhani** [comment]
Yes, Local LLMs can be used, **but cloud-based models like OpenAI's GPT models API are not allowed.**
